tests/functional/extension_functions/test_approximation_functions.py

```diff
@@ -0,0 +1,126 @@
+from collections.abc import Iterable
+from typing import Callable
+
+import duckdb
+import pytest
+from ibis.expr.types.relations import Table
+
+from tests.consumers.acero_consumer import AceroConsumer
+from tests.consumers.duckdb_consumer import DuckDBConsumer
+from tests.functional.common import run_subtrait_on_acero, run_subtrait_on_duckdb
+from tests.functional.extension_functions.testcase_parameters.approximation_tests import (
+    AGGREGATE_FUNCTIONS)
```

Comment:
There's a pretty deep directory structure here and I think we can get something that's more concise and easier to reason about if it gets flattened a bit.

`from tests.functional.approximation_tests import ...` seems more straightforward to me.

Reply:
Thanks, flattened it out a bit.   I thought it was getting a little deep as well.  I wanted a better logical separation for other types of tests later on, but we can re-evaluate when it comes to that point.